packages/template-chakra-storefront/translations/en-US.json

```diff
@@ -295,8 +294,7 @@
     "defaultMessage": "This email already has an account."
   },
   "checkout_confirmation.message.num_of_items_in_order": {
-    "defaultMessage": "{itemCount, plural, =0 {0 items} one {# item} other {# items}}",
-    "description": "# item(s) in order"
+    "defaultMessage": "{itemCount, plural, =0 {0 items} one {# item} other {# items}}"
```

Comment:
We should keep the description since the message here is a bit complex to understand, if you read description, it gives you a better explanation

Reply:
AI makes mistakes sometimes....... fixed